File: apps/bench/ref/sieve.py
import sys
def count_primes(n):
    m=bytearray(n+1); c=0
    for i in range(2,n+1):
        if not m[i]:
            c+=1; m[i*i::i]=b'\1'*len(m[i*i::i])
    return c
print(count_primes(10000000))

File: apps/bench/ref/sieve.js
function countPrimes(n){ const m=new Uint8Array(n+1); let c=0;
  for(let i=2;i<=n;i++) if(!m[i]){c++; for(let j=i+i;j<=n;j+=i) m[j]=1;} return c; }
console.log(countPrimes(10000000));
